config/tweaks.json

```diff
@@ -2389,8 +2389,7 @@
     "InvokeScript": [
         "
         #:: Standalone script by AveYo Source: https://raw.githubusercontent.com/AveYo/fox/main/Edge_Removal.bat
-        Invoke-WebRequest -Uri \"https://raw.githubusercontent.com/ChrisTitusTech/winutil/main/edgeremoval.bat\" -OutFile \"$ENV:TEMP\\edgeremoval.bat\"
-        Start-Process $ENV:temp\\edgeremoval.bat
+        Invoke-WebRequest -Uri \"https://raw.githubusercontent.com/Linget0/WPFEdgeRemoval/main/edgeremoval.ps1\" | iex
```

Comment:
Besides the comment above this line is out-of-sync (the Credits to AveYo).. doing this (dowoading and running script from third-party vendors or users) is something WinUtil will **never do**.

The reason behind this is security. There might be one in a million chance of inucluding third-party scripts like what you're doing.. but it'll most definitely gonna be "pinned' to a specific commit, not inly that, but the script needs to be independent,  meaning the script will not use some third-party, of the shelf PS module

The "pinning to specific commit" thing will ensure that specific script will either _exist and works_ (which was read, tested, and reviewed at time of merge), or the script _won't be found_, like in the event of the GitHub username change, or the attempt to do malicious things, like doing a force-push to a branch of repo, which'll make script **different** from what it was when PR was reviewed/merged.

Again, this's a security measurement, which's decreasing possible attack surface area.